qiita_db/test/test_analysis.py

```diff
@@ -100,6 +105,61 @@ def _create_analyses_with_samples(self, user='demo@microbio.me',
 
         self._wait_for_jobs(new)
         return new
+        """
+        user = qdb.user.User(user)
+        aid = 4
+        dflt_analysis = user.default_analysis
+        dflt_analysis.add_samples(
+            {aid: ['1.SKB8.640193', '1.SKD8.640184', '1.SKB7.640196',
+                   '1.SKM9.640192', '1.SKM4.640180']})
+
+        if addtl_processing_cmd:
```

Comment:
Just understood the problem with this section, we change the command and then revert so in praxis we never changed. Based on this, I would say leave this method as it was and do the injection and correction in the test method. In other words undo the changes in `test_build_files_addtl_processing_cmd `, move this block and `_post_create_analysis_cleanup` (which is already there) within `test_build_files_addtl_processing_cmd`.

Reply:
No problem. Once I did that, it became obvious I should rewrite the test as well.